test/integration/targets/postgresql_user_obj_stat_info/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+# Initial tests of postgresql_user_obj_stat_info module:
+- import_tasks: postgresql_user_obj_stat_info.yml
+  when: postgres_version_resp.stdout is version('10', '>=')
```

Comment:
they can put ```= E'value'``` instead of just ```='value'``` which caused CI failure when we check executed_queries.. nothing else beside that.
It seems that the problem is presented in 9.2 or earlier, not sure. I can try to put 9.4.